### ISSUE TITLE
[BUG] Committing many files does not work

### ISSUE THREAD

**Issue by ZimmerD** [nfdi4plants/ARCitect#201]
**OS and framework information (please complete the following information):**
 - OS: [e.g. Windows 11]
 - ARCitect v. 0.0.35 & 0.0.31

**Describe the bug**
- [x] A clear and concise description of what the bug is
I want to commit 500 files added to my ARC. In the ARCitect, added and changed files are listed [see screenshot 1]. Upon pressing "Commit", the dialogue opens and all steps seem to run successfully [see screenshot 2], however afterwards files are still listed as not committed [see screenshot 3], and sync is not possible due to uncommitted changes [see screenshot 4]. When opening the ARC using Github desktop, I see that the files are indeed not committed. They can be commited using Github desktop or console and then also be synced using the ARCitect.

- [x] Screenshots
![image](https://github.com/nfdi4plants/ARCitect/assets/17741807/dc434498-899a-4d99-9b29-6a6dd429def8)
![image](https://github.com/nfdi4plants/ARCitect/assets/17741807/abcb299d-e868-4067-8f58-5e9558ae6c6c)
![image](https://github.com/nfdi4plants/ARCitect/assets/17741807/4691fe40-f6d1-462d-b6b9-bdc21367635a)
![image](https://github.com/nfdi4plants/ARCitect/assets/17741807/82c631ce-8b13-46d3-b1f7-0b255bbbc8a9)

**Comment by Brilator** [comment]
- try to close all other programs where files of the ARC are opened (esp. Windows Excel, etc.)
https://nfdi4plants.org/nfdi4plants.knowledgebase/docs/guides/git-troubleshooting.html#arc-files-open-in-multiple-programs

**Comment by ZimmerD** [comment]
I just replicated my previous issue and unfortunately closing all other programs which could interact with ARCs files (VSCode etc.) did not fix this. 

I noticed that the first attempted commit using the ARCitect got added to the history and 2 of ~1000+ files got commited. The remaining files cannot be commited using the ARCitect and remain to be listed as uncommited. Again they could be commited using git directly.
